css-flex/css-flexbox_profile/css/styles.css

```diff
@@ -30,15 +30,25 @@ body {
   color: var(--white);
```

Comment:
do not use the universal selector to define a color. 
your "About me" section is not visible because your text color is white like your background color.

Reply:
This line is not visible in VS-Code, so lets see if a the merge will fix it.